components/match2/wikis/rocketleague/match_group_input_custom.lua

```diff
@@ -207,6 +212,45 @@ function matchFunctions.getExtraData(match)
 	return match
 end
 
+function p._getCasterInformation(name, flag, id)
+	if String.isEmpty(flag) then
+		flag = Variables.varDefault(name .. '_flag')
+	end
+	if String.isEmpty(id) then
+		id = Variables.varDefault(name .. '_dn')
+	end
+	if String.isEmpty(flag) or String.isEmpty(id) then
+		local parent = Variables.varDefault(
+			'tournament_parent',
+			mw.title.getCurrentTitle().text
+		)
+		local pageName = mw.ext.TeamLiquidIntegration.resolve_redirect(name)
+		local data = mw.ext.LiquipediaDB.lpdb('broadcasters', {
+			conditions = '[[page::' .. pageName .. ']] AND [[parent::' .. parent .. ']]',
+			query = 'flag, id',
+			limit = 1,
+		})
+		if type(data) == 'table' and data[1] then
+			flag = String.isEmpty(flag) and data[1].flag or flag
+			id = String.isEmpty(id) and data[1].id or id
+		end
+	end
+	if String.isNotEmpty(flag) then
+		Variables.varDefine(name .. '_flag', flag)
+	end
+	if String.isEmpty(id) then
+		id = name
+	end
+	if String.isNotEmpty(id) then
+		Variables.varDefine(name .. '_dn', id)
+	end
+	return {
+		name = name,
```

Comment:
This is pagename not name, right?

Reply:
yes it is pagename
displayname is `id` in this function

Reply:
Probably should be named as such